src/WordPress/Blueprints/Runner/Step/MkdirStepRunner.php

```diff
@@ -2,16 +2,16 @@
 
 namespace WordPress\Blueprints\Runner\Step;
 
+use Symfony\Component\Filesystem\Exception\IOException;
+use WordPress\Blueprints\BlueprintException;
 use WordPress\Blueprints\Model\DataClass\MkdirStep;
 
 
 class MkdirStepRunner extends BaseStepRunner {
 
-	function run( MkdirStep $input ) {
-		// @TODO: Treat $input->path as relative path to the document root (unless it's absolute)
-		$success = mkdir( $input->path );
-		if ( ! $success ) {
-			throw new \Exception( "Failed to create directory at {$input->path}" );
-		}
+    function run(MkdirStep $input) {
+        $path = $input->path;
+        $this->fileManager->assertNoFileExists($path);
```

Comment:
This line obscures that `$path` is relative to the document root and needs to be resolved.